docs/components/integration-framework/connectors/out-of-the-box-connectors/googledrive.md

```diff
@@ -0,0 +1,245 @@
+---
+id: googledrive
+title: Google Drive Connector
+description: Create empty folders or documents from templates from your BPMN process.
```

Comment:
Pls update text to:
"Create folders or files from a Google Drive template from your BPMN process."

Reply:
Addressed in a new revision. Will publish shortly.